core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -144,6 +144,17 @@
 
   private int maxInboundMessageSize = GrpcUtil.DEFAULT_MAX_MESSAGE_SIZE;
 
+  /**
+   * If true, indicates that the transport may use the GET method for RPCs, and may include the
+   * request body in the query params.
+   */
+  boolean useGetForSafeMethods;
```

Comment:
Since these aren't used by this builder, let's not add them here. If we add methods in the future to set these it should probably be per-transport. We can just hard-code false instead of calling methods like `useGetForSafeMethods()`. If you want it for the readability, make a local variable like `boolean useGetForSafeMethods = false;` and then pass useGetForSafeMethods as an argument.

It's fine to have the plumbing like you added to the ClientTransportFactory implementations.

Reply:
I moved ti down to the individual transport builders.